local-cli/utils/buildAndDeployUtils.js

```diff
@@ -36,13 +30,13 @@ function launchServer(options) {
   const opts = {
     cwd: options.root,
     detached: true,
-    stdio: 'ignore'
+    stdio: options.verbose ? 'inherit' : 'ignore'
   };
```

Comment:
Not sure what the impact of launching `cmd.exe` with `stdio: 'inherit'`. I assume this only applies to the `stdio` of the initial cmd call, not the script that ultimately runs (the packager window should retain control of its own output)

Reply:
The old code used `'inherit' : 'ignore'` for windows, and just `'ignore'` for wpf.

After looking into it today, I believe you are correct.